message_test.go

```diff
@@ -391,6 +391,15 @@ func TestAddressList(t *testing.T) {
 	}
 }
 
+func TestEmptyAddressList(t *testing.T) {
+    addrs := make([]*Address, 0)
```

Comment:
Style: we use tabs.

You can make sure the style is okay by running `go fmt ./...`.

Apart from this LGTM!

Reply:
(forgot to run gofmt). I'll try to build hydroxide against this to see if the issue is properly fixed